cli/src/main/java/com/liferay/blade/cli/command/ServerStartArgs.java

```diff
@@ -23,7 +23,9 @@
  * @author Gregory Amerson
  * @author Simon Jiang
  */
-@Parameters(commandDescription = "Start server defined by your Liferay project", commandNames = "server start")
+@Parameters(
+	commandDescription = "Start server defined by your Liferay project", commandNames = {"server start", "server run"}
```

Comment:
Lets not do this.  Lets create two separate classes `ServerStartCommand` and `ServerRunCommand`.  Then we don't need the `getArgs()` command.  

Also when we do 'server start' then we need to print out a helpful message about how they can "stop" the server or "tail the log" in the case of tomcat.